command/base.go

```diff
@@ -271,32 +278,15 @@ func (c *BaseCommand) validateMFA(reqID string, methodInfo MFAMethodInfo) int {
 
 	client, err := c.Client()
 	if err != nil {
-		c.UI.Error(err.Error())
-		return 2
+		return nil, err
 	}
 
 	secret, err := client.Sys().MFAValidate(reqID, mfaPayload)
```

Comment:
we can probably just return this and get rid of the logic below

Reply:
Fixed in my local branch

Reply:
Agreed